po/extra/meson.build

```diff
@@ -0,0 +1,3 @@
+i18n.gettext(meson.project_name() + '-extra',
+    args: '--directory=' + meson.source_root()
```

Comment:
I think we could go for an `install: false` here. The translations for `extra` get merged into the `.desktop` file so we don't need to install the compiled translations.